### PR TITLE
Docs: 

### PR DESCRIPTION
**Pull request by tpdorsey**
## Summary

Doc links and guidance for GA.

## Changes

Updates landing page with guidance to find relevant version of docs.

Updates hard links throughout for compatibility with new hosted URLs

## Checklist

This PR:

- [ ] adds new tests (if appropriate)
- [ ] adds a change file in the `changes/` directory (if appropriate)
- [ ] updates docstrings for any new functions or function arguments, including `docs/outline.toml` for API reference docs (if appropriate)

### CODE REVIEW

File: src/prefect/core/flow.py [hunk 1]
```diff
@@ -1666,7 +1666,7 @@ def register(
                 prior to serialization; defaults to `True`
             - labels (List[str], optional): a list of labels to add to this Flow;
                 useful for associating Flows with individual Agents; see
-                http://docs.prefect.io/orchestration/agents/overview.html#labels
+                http://docs.prefect.io/v2orchestration/agents/overview.html#labels
```
Comment (anna-geller) [inline src/prefect/core/flow.py:1669]:
```suggestion
                http://docs.prefect.io/v2/orchestration/agents/overview.html#labels
```

File: docs/orchestration/getting-started/more-resources.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ examples, run the following:
 $ prefect create project "Prefect Examples"
 
 # Register all the examples into the "Prefect Examples" project
-$ prefect register --json https://docs.prefect.io/examples.json --project "Prefect Examples"
+$ prefect register --json https://docs.prefect.io/v2examples.json --project "Prefect Examples"
```
Comment (anna-geller) [inline docs/orchestration/getting-started/more-resources.md:35]:
```suggestion
$ prefect register --json https://docs.prefect.io/v2/examples.json --project "Prefect Examples"
```

**Review by anna-geller — APPROVED** [review]
LGTM - added two /, that's all :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -209,7 +209,7 @@ Released on February 23, 2022.
 
 - Authentication with tokens has been removed; use API keys instead. - [#4643](https://github.com/PrefectHQ/prefect/pull/4643)
 - Python 3.6 is no longer supported; use Python 3.7+ instead. - [#5136](https://github.com/PrefectHQ/prefect/pull/5136)
-- Flow `Environment`s have been removed; use `RunConfig`s instead. - [#5072](https://github.com/PrefectHQ/prefect/pull/5072), [docs](https://docs.prefect.io/orchestration/flow_config/upgrade.html)
+- Flow `Environment`s have been removed; use `RunConfig`s instead. - [#5072](https://github.com/PrefectHQ/prefect/pull/5072), [docs](https://docs.prefect.io/v2/orchestration/flow_config/upgrade.html)
```
Comment (billpalombi) [inline CHANGELOG.md:212]: Many, if not all, of the places that I see `v2`, it should in fact be `v1`, no?

**Review by billpalombi — CHANGES_REQUESTED** [review]
I may be mistaken, but I think most of these URLs should be `v1`

File: CHANGELOG.md (hunk 3 above)
Comment (tpdorsey) [reply CHANGELOG.md:212]: Yes you are correct. I'll fix these up

**Comment by tpdorsey** [comment]
Closed in favor of https://github.com/PrefectHQ/prefect/pull/6052

**Pull request closed without merging**